content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -32,6 +32,26 @@ resources, such as different versions of the same software: use
 {{< glossary_tooltip text="labels" term_id="label" >}} to distinguish
 resources within the same namespace.
 
+{{< note >}}
+For a production cluster, consider _not_ using the `default` namespace. Instead, make other namespaces and use those.
+{{< /note >}}
+
+## Initial namespaces
+
+Kubernetes starts with four initial namespaces:
+
+`default`
+: Kubernetes includes this namespace so that you can start using your new cluster without first creating a namespace.
```

Comment:
May be implied, but you could say that any Pods deployed without an identified namespace automatically go into the `default` namespace.

Reply:
Fair enough.